arduino/IoTkit/examples/IoTKitQuickStartExample/Setup.ino

```diff
@@ -0,0 +1,262 @@
+#include <stdio.h>
+#include <IoTkit.h>
+#include <Ethernet.h>
+#include <aJSON.h>
+#include <rgb_lcd.h>  //library from https://github.com/Seeed-Studio/Sketchbook_Starter_Kit_V2.0/tree/master/libraries/Grove_LCD_RGB_Backlight
+#include <Wire.h>
+#include <stdio.h>
+
+char  * networkSSID="CHANGE_ME";
+char  * WIFI_identity="CHANGE_ME";
+char  * WIFI_password="CHANGE_ME";
+char  * activationCode="CHANGE_ME";
+
+IoTkit iotkit;
+int inPinTemperatureSensor = A0;
+int outPinLed = 3;
+rgb_lcd lcd;
+
+float TemperatureValue;
+float minTempBlue = 22;//0% value
```

Comment:
After last demo, I think you can make range wider, making lower bound lower, e.g. 20.
When temperature was less than 22, display was red (hot) instead of blue (cold).